generate.go

```diff
@@ -27,7 +29,6 @@ var generateFlags = []cli.Flag{
 	cli.StringSliceFlag{Name: "groups", Usage: "supplementary groups for the process"},
 	cli.StringSliceFlag{Name: "cap-add", Usage: "add capabilities"},
 	cli.StringSliceFlag{Name: "cap-drop", Usage: "drop capabilities"},
-	cli.StringFlag{Name: "cgroup", Usage: "cgroup namespace"},
```

Comment:
You probably didn't intend to remove this (it landed in #122).  Maybe a busted rebase?